connector-runtime/connector-runtime-core/src/test/java/io/camunda/connector/runtime/core/outbound/AnnotatedFunction.java

```diff
@@ -23,7 +23,8 @@
 @OutboundConnector(
     name = "ANNOTATED",
     inputVariables = {"a", "b"},
-    type = "io.camunda:annotated")
+    type = "io.camunda:annotated",
+    timeout = 20000)
```

Comment:
Can you incorporate this change without changing the annotation? I would like to avoid runtime configuration in the code and rather make use the application config (environment).

Reply:
I could. However, I was thinking whether it could make sense to have the timeout configurable somewhere on the connector side.

Maybe by rephrasing it from timeout to duration or invocationDuration?

Reply:
I think we should separate the runtime behavior form the code and therefore not introduce another field in the annotation.

Reply:
Alright. In this case, the only way to configure the timeout will be via env variable